sdk/core/core-tracing/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0-preview.8 (Unreleased)
+
+- Fix tracingPolicy() to set standard span attributes [PR link](https://github.com/Azure/azure-sdk-for-js/pull/6565).  Now the following are set correctly for the spans
+  - `http.method`
+  - `http.url`
+  - `http.user_agent`
+  - `http.status_code`
+  - `requestId`
+  - `serviceRequestId`
+  - `kind` = Client
```

Comment:
These two lines seem a little mangled?

Reply:
I tried to copy from the issue linked in the PR and add some formatting for the `kind`. It is not fenced in the linked issue, nor is the span name.